src/docker-credential-acr/acr_login.go

```diff
@@ -71,7 +71,7 @@ func GetUsernamePassword(serverAddress string, identityToken string) (user strin
 		var challenge *authDirective
 		if challenge, err = receiveChallengeFromLoginServer(serverAddress); err != nil {
 			// ignore all error when receiving the challenge
-			logrus.Warnf("[Azure Login Helper] server %s does not responded with a valid challenge, reverting to default login...\nerror: %s\n", serverAddress, err)
+			logrus.Infof("[Azure Login Helper] server %s does not responded with a valid challenge, reverting to default login...\nerror: %s\n", serverAddress, err)
```

Comment:
FYI, I have now fixed the typo (see file changed tab) but somehow the PR comment section didn't show it's fixed.